cmd/tailscaled/install_darwin.go

```diff
@@ -73,9 +73,18 @@ func uninstallSystemDaemonDarwin(args []string) (ret error) {
 		}
 	}
 
-	err = os.Remove(sysPlist)
-	if os.IsNotExist(err) {
-		err = nil
+	if err := os.Remove(sysPlist); err != nil {
+		if os.IsNotExist(err) {
+			err = nil
+		}
+		if ret == nil {
+			ret = err
+		}
+	}
+	if err := os.Remove(targetBin); err != nil {
```

Comment:
I was afraid of removing this, in case we weren't the ones who installed it. I was thinking of setting some resource fork metadata on the file to mark it as the one owned by us, but in retrospect that's probably overkill. This isn't sensitive data. If they ran this command they can just re-install it too.